jalien.sh

```diff
@@ -0,0 +1,35 @@
+package: JAlien
+version: "%(tag_basename)s"
+tag: master
+source: https://gitlab.cern.ch/jalien/jalien.git
+requires:
+ - Java
+ - JAlien-XRootD
+build_requires:
+ - Java
+---
+#! /bin/bash -e
```

Comment:
Spaces between `#!` and the interpreter [are allowed](https://unix.stackexchange.com/questions/276751/is-space-allowed-between-and-bin-bash-in-shebang) but we don't use them, please remove for consistency.